solutions/usaco-991.mdx

```diff
@@ -92,7 +92,7 @@ low = 1
 high = n + 1
 
 while low + 1 < high:
-	middle = low + high >> 1
+	middle = (low + high) // 1
```

Comment:
wait what
you're dividing by 1?

Reply:
mybad i just committed aadit's suggestion.. i assumed it was correct

Reply:
changed it back

Reply:
or just
```suggestion
	middle = (low + high) // 2
```